src/App.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import FunctionComponent from './FunctionalComponent/FunctionalComponent';
+import ReactComponent from './ReactComponent/ReactComponent';
+import CreateElement from './CreateElement/CreateElement';
+import PureComponentContainer from './PureComponentContainer/PureComponentContainer';
+import './App.css';
+
+function App() {
+  return (
+    <div className="container">
+      <div className="wrapper">
+      <FunctionComponent></FunctionComponent>
+      <ReactComponent></ReactComponent>
+      <CreateElement></CreateElement>
+      <PureComponentContainer></PureComponentContainer>
```

Comment:
FYI, if your component does not take child elements it's possible to have a shorthand syntax:

```jsx
<PureComponentContainer />
```

Reply:
used short hand syntax